var/spack/repos/builtin/packages/coin3d/package.py

```diff
@@ -7,17 +7,24 @@
 from spack.pkg.builtin.boost import Boost
 
 
-class Coin3d(AutotoolsPackage):
+class Coin3d(AutotoolsPackage, CMakePackage):
     """Coin is an OpenGL-based, 3D graphics library that has its roots in the
     Open Inventor 2.1 API, which Coin still is compatible with."""
 
     homepage = "https://github.com/coin3d/coin"
-    url = "https://github.com/coin3d/coin/archive/Coin-4.0.0.tar.gz"
+    url = "https://github.com/coin3d/coin/releases/download/Coin-4.0.0/coin-4.0.0-src.tar.gz"
 
+    version("4.0.0", sha256="e4f4bd57804b8ed0e017424ad2e45c112912a928b83f86c89963df9015251476")
     version("3.1.0", sha256="70dd5ef39406e1d9e05eeadd54a5b51884a143e127530876a97744ca54173dc3")
     version("3.0.0", sha256="d5c2eb0ecaa5c83d93daf0e9e275e58a6a8dfadc74c873d51b0c939011f81bfa")
     version("2.0.0", sha256="6d26435aa962d085b7accd306a0b478069a7de1bc5ca24e22344971852dd097c")
 
+    build_system(
+        conditional("cmake", when="@4.0.0:"),
+        conditional("autotools", when="@:3.1.0"),
```

Comment:
```suggestion
        "autotools",
```
Autotools is still supported in the latest release

Reply:
eh, but it doesn't work out of the box so i let it be.